terraform/statcan.alpha.canada.ca.tf

```diff
@@ -13,7 +13,27 @@ resource "aws_route53_record" "statistique-alpha-canada-ca-CNAME" {
     name    = "statistique.alpha.canada.ca"
     type    = "CNAME"
     records = [
-        "www.prod.cloud.statcan.ca"
+        "www-alpha.prod.cloud.statcan.ca"
+    ]
+    ttl     = "300"
+}
+
+resource "aws_route53_record" "energy-information-alpha-canada-ca-CNAME" {
+    zone_id = aws_route53_zone.alpha-canada-ca-public.zone_id
+    name    = "energy-information.alpha.canada.ca"
+    type    = "CNAME"
+    records = [
+        "www-alpha.prod.cloud.statcan.ca"
+    ]
+    ttl     = "300"
+}
+
+resource "aws_route53_record" "information-energie-alpha-canada-ca-CNAME" {
+    zone_id = aws_route53_zone.alpha-canada-ca-public.zone_id
+    name    = "statistique.alpha.canada.ca"
```

Comment:
Looks like this should be the french version of energy-information.alpha.canada.ca

Reply:
Good catch! That should definitely be information-energie.alpha.canada.ca

Reply:
@maxneuvians Fix this up tommorow and it's good to go.